t2/chess_main.py

```diff
@@ -0,0 +1,32 @@
+'''Напишите функцию в шахматный модуль. Используйте генератор случайных чисел для случайной расстановки ферзей в задаче выше.
+Проверяйте различный случайные варианты и выведите 4 успешных расстановки.'''
+import random
+
+from t2 import chess_module
+
+BOARD_SIZE = 8
+NUMBER_OPTIONS = 4
+
+
+def random_qeen_list(size):
+    res = []
+    i = 0
+    while i < size:
+        qeen = [random.randint(0, size), random.randint(0, size)]
```

Comment:
У вас генерация происходит от 0 до 8 включительно, а должна от 1 до 8
```suggestion
        qeen = [random.randint(1, size), random.randint(1, size)]
```